### DISCUSSION TITLE
feat(env): opt-out?

### DISCUSSION THREAD

**Discussion by RyKilleen** [pnpm/pnpm#4531] [Q&A]
Is there any intent to allow the disabling of `pnpm env`? It's currently conflicting with the usage of other node version managers.

**Comment by zkochan** [comment]
Why do you need to disable it? It is not active by default. You should use the `use-node-version` setting or run `pnpm env`. Otherwise, it is not used.

**Reply by RyKilleen** [reply]
It definitely appears to be in use for me.

```
$ fnm list
* v16.14.2 default
* system
```

Trying to run a command results in this output:
` WARN  Unsupported engine: wanted: {"node":">=16"} (current: {"node":"v14.17.0","pnpm":"7.0.0"})`

This only resulted after re-installing `pnpm`. Happy to provide more diagnostics or info if you know what will help guide me here, or convert into an issue.
